ethcore/client-traits/src/lib.rs

```diff
@@ -397,30 +397,69 @@ pub trait BlockChainClient:
 
 	/// Schedule state-altering transaction to be executed on the next pending block.
 	fn transact_contract(&self, address: Address, data: Bytes) -> Result<(), transaction::Error> {
```

Comment:
I'd remove that method now, it's really easy to use `transact` instead.

Reply:
I removed it. :+1:

Reply:
Perfect, thanks!